src/posix/platform/radio.cpp

```diff
@@ -50,6 +52,8 @@ static ot::Spinel::RadioSpinel<ot::Posix::SpiInterface, RadioProcessContext> sRa
 #error "OPENTHREAD_POSIX_CONFIG_RCP_BUS only allows OT_POSIX_RCP_BUS_UART and OT_POSIX_RCP_BUS_SPI!"
 #endif
 
+static ot::Posix::MaxPowerTable sMaxPowerTable;
```

Comment:
Do we need to move line 55 and 37 between `#if OPENTHREAD_POSIX_CONFIG_MAX_POWER_TABLE_ENABLE` and `#endif` ?